build.fsx

```diff
@@ -258,8 +254,8 @@ Target.create "GenerateHelpDebug" (fun _ ->
 Target.create "KeepRunning" (fun _ ->
     use watcher = 
         
-        !! "docsrc/content/**/*.*" 
-        |> ChangeWatcher.run (fun changes ->
+        Fake.FileSystem.(!!) "docsrc/content/**/*.*" 
+        |> Fake.ChangeWatcher.WatchChanges (fun changes ->
```

Comment:
This was actually an oversight. We failed to add the new ChangeWatcher-API to `FakeLib.dll`